controllers/signup.go

```diff
@@ -0,0 +1,71 @@
+package controllers
+
+import (
+	"log"
+	"net/http"
+	"regexp"
+
+	"github.com/UniversityRadioYork/2016-site/models"
+	"github.com/UniversityRadioYork/2016-site/structs"
+	"github.com/UniversityRadioYork/2016-site/utils"
+	"github.com/UniversityRadioYork/myradio-go"
+)
+
+// SignUpController is the controller for processing signup requests.
+type SignUpController struct {
+	Controller
+}
+
+// NewSignUpController returns a new SignUpController with the MyRadio
+// session s and configuration context c.
+func NewSignUpController(s *myradio.Session, c *structs.Config) *SignUpController {
+	return &SignUpController{Controller{session: s, config: c}}
+}
+
+// Get handles the HTTP GET request r for the get involved, writing to w.
+func (gic *SignUpController) Get(w http.ResponseWriter, r *http.Request) {
+	formParams := r.URL.Query()
+	var feedback []string
+
+	//Validate that necessary params are present and correct(enough)
+	if formParams["fname"][0] == "" {
+		feedback = append(feedback, "You need to provide your first name")
+	}
+	if formParams["sname"][0] == "" {
+		feedback = append(feedback, "You need to provide your second name")
```

Comment:
Probably want to be "Last Name" to be consistent. Probably capitalised like the form labels :)